src/js/string-class.js

```diff
@@ -74,68 +78,72 @@ const STRING_EXTENSION = {
   },
 
   /**
-   * wordCount method returns the number of
-   * words in the string
+   * wordCount
+   *
+   * returns the total number of words in the string
    *
-   * @return {Int} number
+   * @return {Int} integer
    */
   wordCount() {
     return this.words().length;
   },
 
   /**
-   * toCurrency method returns the currency
-   * representation of the string
+   * toCurrency
    *
-   * @return {String} string
+   * returns the currency representation of the string
+   *
+   * @return {String} stirng
    */
   toCurrency() {
-    const left = this.split('.')[0].replace(/[^0-9]/g, '').reverse()
-      .replace(/(\d{3})/g, '$&,')
-      .replace(/,$/, '')
-      .reverse();
-
-    const right = this.split('.')[1] || '00';
-    return `${left}.${right}`;
+    return Number(this)
+      ? Number(this).toFixed(2).replace(/(\d)(?=(\d{3})+\.)/g, '$1,')
+      : 'Error: Invalid input';
   },
 
   /**
-   * fromCurrency method returns the number
-   * representation of the string
+   * fromCurrency
+   *
+   * returns the number representation of the string
    *
    * @return {Number} number
    */
   fromCurrency() {
-    return this.replace(/,+/g, '');
+    return Number(this.replace(/,/g, '')) || 'Error: Invalid input';
   },
 
   /**
-   * inverseCase method returns each letter of the
-   * string as an inverse of its current case
+   * inverseCase
+   *
+   * returns each letter of the string as an inverse of its current case
    *
    * @return {String} string
    */
   inverseCase() {
-    return this.replace(/[a-zA-Z]/g, (ch) => {
-      return /[a-z]/.test(ch) ? ch.toUpper() : ch.toLower();
+    return this.replace(/[a-zA-Z]/g, (character) => {
+      if (/[a-z]/.test(character)) return character.toUpper();
+      return character.toLower();
     });
   },
 
   /**
-   * alternatingCase method returns each letter of the
-   * string in alternating cases
+   * alternatingCase
+   *
+   * returns each letter of the string in alternating cases
    *
    * @return {String} string
    */
   alternatingCase() {
-    return this.replace(/[a-zA-Z]/g, (ch, index) => {
-      return index % 2 === 0 ? ch.toLower() : ch.toUpper();
+    return this.replace(/[a-zA-Z]/g, (character, index) => {
+      if (index % 2 === 0) return character.toLower();
```

Comment:
Verbose implementation. Use ternary operator instead.
example.
instead of writing 

```
if (condition) return value;
return defaultValue;
```

you could do something like this

`return condition ? value : defaultValue;`